### DISCUSSION TITLE
Thank you @tuchida!

### DISCUSSION THREAD

**Discussion by p-bakker** [mozilla/rhino#1607] [Show and tell]
It seems we've lost @tuchida as a participant in the Rhino project: He closed PR #1537 and issue #1554 approximately 3 weeks ago and it seems he deleted his fork of the Rhino repository 😐

@tuchida's contributions to this project span almost 12 years, contributing, amongst many other things, our BigInt and for-of loop implementations and recently cleaning up the IRFactory impl. and a lot of nitty-gritty fixes, all in all making him the [#7](https://github.com/mozilla/rhino/graphs/contributors) contributor to this project at this time

Unsure about his reasonings, I'd like to hereby thank him for his valuable contributions to this project over this very long time!

I'm gonna reopen PR #1537 and issue #1554, because I think both are still very valuable

**Comment by gbrail** [comment]
Definitely one of the people who has done the most to make Rhino viable
again. Thanks!

**Reply by p-bakker** [reply]
@gbrail any chance you can reopen PR #1537, cause I can't. Not sure if that's some permission thing or if there's something else going on
